scripts/hud/customizer.ts

```diff
@@ -1,15 +1,5 @@
-// I'm trying something new here and using a namespace to escape the namespace pollution nightmare currently have
-// In short, TypeScript seems to put stuff that doesn't include `import`/`export`s in the global namespace,
-// and using those is impossible since pure V8 doesn't have an import/module system.
-
-// The below `Axis` enum is breaking compiles because we have some fucking JSDOC (!) typedef in linegraph.js
-// that overlaps with this one. So, I'm using a namespace (which just gets compiled to an IIFE for scope isolation)
-// and not using a static class - that was essentially just playing the role of an IIFE anyway!
-
-// Not sure if this is what we'll stick with it, but seems good for now.
-
-// Sasha: had to change some types from Panel to GenericPanel
-
+import { values } from 'util/enum';
```

Comment:
Nit: I'd do `import * as Enum` here so you can do `Enum.values`, otherwise function's usage is a bit unclear in code. That was my intention when writing those utils, getting something very similar to `Object.values`, `Object.keys` etc...

Reply:
Addressed in 74f396aa55355f1288479bab2290b85024d24fa6